content/post/reporting-is-a-gateway-drug.md

```diff
@@ -0,0 +1,70 @@
+---
+author: Scott Breitenother
+title: "Reporting is a Gateway Drug"
+publishDate: "2018-04-29"
+draft: true
+tags: 
+ - analytics
+ - stakeholders
+ - reporting
+---
+
+I love doing reporting. Well I don't actually love doing reporting, but I love what it can do for an Analytics team. If executed well, reporting can be the gateway drug, resulting in an organization that is completely addicted to its Analytics team. If executed poorly, the Analytics team can turn into a team of reporting monkeys - we all know what that is like. Here is some advice on how to use reporting as a means to create strong stakeholder relationships in your organization.
+<!--more-->
+
+
+## Setting the scenario
+You are starting an Analytics function from scratch and any of the following is true:
+* You have no idea what an Analytics team should do
+* You are still learning about your company's business model
+* You have a limited technical skill set
+* You have no analytics infrastructure
+
+Extra points if your organization knows analytics is important, but does not know exactly what the Analytics team should do.
+
+Don't panic or run for the door - take a long around. You may notice there are quite a few regular (daily, weekly, monthly) reports, typically compiled in Google Sheets or Excel. These are not simple copy and paste jobs or automated scripts that need to be run, these are complex, manual process that take up a non-negligible chunk someone's time. The source data usually sits in several different places and the transformations involve calculations, business logic and personal opinion. There is probably little to no documentation on the process and it is all in someone's head. And I know what you are thinking - no you can't just get rid of the reports - these reports are critical to running the business (e.g., what were sales yesterday). However, there may be future opportunities to optimize / rationalize the reports.
+
+
+## Steps to success
+So how do you take reporting lemons and make analytics lemonade?
+1. Same report, but faster
+2. Evolve the report
+3. Add insights
+
+
+### 1. Same report, but faster
+You may want to come in, analytics guns blazing and transform everything day one. Let me tell you why this is a terrible idea - your stakeholders don't know you (and you probably don't know them). This is how it will go:
+
+You: Hi, I am in Analytics. I just did XYZ analysis and you should now measure yourself against this new KPI I created
+Stakeholder: Um... no
```

Comment:
Alternative:

Stakeholder: Um... thanks but no thanks. I'm very busy and it seems like this is about to create a bunch more work for me.